src/Sulu/Bundle/AdminBundle/Resources/js/components/DatePicker/README.md

```diff
@@ -0,0 +1,71 @@
+The DatePicker component builds on the top of following component: 
+https://github.com/YouCanBookMe/react-datetime
```

Comment:
It reads a little bit nicer if you use an actual link built into the sentence instead: something like `\[link name\](url)`